packages/primitives/src/ListItem/ListItem.tsx

```diff
@@ -76,12 +76,12 @@ export const listItemRecipe = sva({
         root: {
           '&[aria-current="true"], &[aria-current="page"]': {
             background: "var(--background-current)",
-            borderBlockStartColor: "transparent",
-            borderBlockEndColor: "transparent",
+            borderInlineWidth: "6px",
```

Comment:
3xsmall

Reply:
Akkurat her er det noe galt, det virker ikke som den tar in de semantiske verdiene.
Det funker om du tar inn padding, men ikke akkurat med borderInlineWidth.

Reply:
Stemmer det. Ignorer denne!